gc.c

```diff
@@ -45,7 +45,8 @@ void *GC_thread(void *val)
 			if(config.debug & DEBUG_GC) timer_start(GC_TIMER);
 
 			int removed = 0;
-			if(config.debug & DEBUG_GC) logg("GC starting, mintime: %lu %s", mintime, ctime(&mintime));
+			char ctimebuffer[32]; // need at least 26 bytes according to ctime_r man page
+			if(config.debug & DEBUG_GC) logg("GC starting, mintime: %lu %s", mintime, ctime_r(&mintime, ctimebuffer));
```

Comment:
`ctime_r` seems to be marked obsolete:
> POSIX.1-2008 marks asctime(), asctime_r(), ctime(), and ctime_r() as obsolete, recommending the use of strftime(3) instead.

https://linux.die.net/man/3/ctime_r

Reply:
I replaced `ctime` by our own routine we already use when generating the string representation for the `pihole-FTL.log` file